pkgs/by-name/mo/molden/package.nix

```diff
@@ -54,7 +54,7 @@ stdenv.mkDerivation rec {
     homepage = "http://www3.cmbi.umcn.nl/molden/";
     license = {
       fullName = "Free for academic/non-profit use";
-      url = "http://www3.cmbi.umcn.nl/molden/CopyRight.html";
+      url = "https://www3.cmbi.umcn.nl/molden/CopyRight.html";
```

Comment:
I changed a link to the license page by accident here. The HTTP link takes you to a 404 page, so maybe it's not a huge deal. I can fix it in a follow up commit before they all get squashed together.